source/dataman/AMScanDictionary.cpp

```diff
@@ -218,11 +218,8 @@ QString AMScanDictionary::krFacilityName(const QString& arg) {
 	if(!scan_)
 		return "[??]";
 
-	AMRun r;
-	r.loadFromDb(scan_->database(), scan_->runId());
-
 	AMFacility f;
-	f.loadFromDb(scan_->database(), r.facilityId());
+	f.loadFromDb(scan_->database(), 1); //AMFacility table will have only one entry, which is the definition of this facility
```

Comment:
Pedantically, this is not correct.  Wouldn't you want to get the id of the facility from the name (it is technically possible to have multiple facilities in a database (especially true with the current BioXAS setup I think).